crates/sui-adapter-transactional-tests/tests/programmable/coin_negative.move

```diff
@@ -23,8 +23,10 @@ module test::fake {
 //> 0: sui::coin::mint<test::fake::FAKE>(Input(0), Input(1));
 //> TransferObjects([Result(0)], Input(2))
 
-//# programmable --sender A --inputs object(2,0) 2
+//# programmable --sender A --inputs object(2,0) 2 @A
 //> SplitCoins(Input(0), [Input(1)]);
+//> TransferObjects([Result(0)], Input(2))
```

Comment:
These small test changes with an added transfer to not have unused values without drop